board/piksiv3/rootfs-overlay/etc/init.d/S86network_daemon

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+name="network_daemon"
+#cmd="network_daemon --interface eth0"
```

Comment:
What's the use case for tracking a single interface?

Reply:
Initially I believe we are only interested in the cell modem usage, this would act as a crude filter. At this point I have the filtering being done at the client side but it's not clear to me which is a more optimal solution.

Reply:
I think filtering on the device side would make more sense at the moment (so we can reduce the amount of data coming out of device), maybe this could be configurable (remotely) later if we want stats on other interfaces.

Reply:
Agreed, I can remove the line if that is more clear.